contents/blog/posthog-vs-growthbook.md

```diff
@@ -0,0 +1,637 @@
+---
+title: "In-depth: PostHog vs GrowthBook"
+date: 2023-09-29
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+---
+
+PostHog and GrowthBook share big similarities. Both provide open-source, self-serve feature flags and experimentation. Beyond this, there are big differences:
+
+- GrowthBook is a warehouse-native feature flag and experiments. It focuses on integrating with the product and data tools you already use.
+
+- PostHog is an all-in-one product and data platform. Beyond [feature flags](/docs/feature-flags) and [experiments](/docs/experiments), it includes [product analytics](/docs/product-analytics), [session replay](/docs/session-replay), [surveys](/docs/surveys), [CDP](/docs/cdp), and more.
+
+This post goes over their differences, platforms, features, pricing, reporting, integrations, and frequently asked questions.
+
+## How is PostHog different?
+
+### 1. It is an all-in-one platform
+
+PostHog brings all the tools engineers need for testing, releasing, and measuring the success of new features. Feature flags and A/B testing are only part of a suite of tools PostHog offers.
+
+![PostHog homepage features](../images/blog/posthog-vs-growthbook/build.png)
+
+PostHog combines usage, performance, and behavioral data with flags and experiments. PostHog’s [data warehouse](/docs/data-warehouse) also enables you to pull in data from external sources (like GrowthBook is reliant on).
+
+Having all these product and data tools together enables you to do better analysis of shipped features and make better decisions about what you are building next. 
+
+### 2. Built for startups and engineers
```

Comment:
Small thing, but I'd change each of these to be a benefit: PostHog is built for startups and engineers / PostHog is easier to setup, etc.